mmaction/datasets/ssn_dataset.py

```diff
@@ -0,0 +1,660 @@
+import copy
+import math
+import os.path as osp
+
+import mmcv
+import numpy as np
+from torch.nn.modules.utils import _pair
+
+from mmaction.localization.proposal_utils import temporal_iou
+from .base import BaseDataset
+from .registry import DATASETS
+from .utils import (load_localize_proposal_file, parse_directory,
+                    process_localize_proposal_list)
+
+
+class SSNInstance(object):
+    """Proposal instance of SSN.
+
+    Args:
+        start_frame (int): Index of the proposal's start frame.
+        end_frame (int): Index of the proposal's end frame.
+        num_frames (int): Total frames of the video.
```

Comment:
`num_frames` is a little bit ambiguous. do you have other suggestions? how about `num_video_frames`